Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -93,6 +94,11 @@ python3 create-external-cluster-resources.py --upgrade --rbd-data-pool-name repl
     An existing non-restricted user cannot be converted to a restricted user by upgrading.
     The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
 
+#### Multi Site support
```

Comment:
for external clusters it was not required to create Mulitsite CRs, it will add a lot of complexity to the rook layer. Since these resources are not created by Rook at the backend and are very difficult to manage. So please remove this section

Reply:
It is needed otherwise how rook will always consider it as default here,
https://github.com/rook/rook/blob/45e3ca94bffe90700c339758aabf138c2f184fbf/pkg/operator/ceph/object/objectstore.go#L206

Reply:
have u seen failures by having default values for the external cluster?

Reply:
removed it